projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -15,8 +15,9 @@
     width: auto !important;
   }
 
-  ::ng-deep tr > td.mat-cell:not(:first-of-type):not(.mat-column-actions) {
-    padding: 0 8px;
+  ::ng-deep tr > td.mat-cell:not(:first-of-type):not(.mat-column-actions),
+  ::ng-deep tr > th.mat-header-cell:not(:first-of-type):not(.mat-column-actions) {
+    padding: 8px;
```

Comment:
Is this change required?
As far as I remember original line that was removed, was the part of the design fixes